tests/context.py

```
@@ -90,12 +90,12 @@
 
 try:
-	if 'pak' not in sys.modules:
-		import pak
+	if 'pypak' not in sys.modules:
+		import pypak
 	else:  # pragma: no branch
-		pak = sys.modules["""pak"""]
+		pypak = sys.modules["""pypak"""]
 except Exception:  # pragma: no branch
-	raise ImportError("[CWE-440] Python PAK Repo Failed to import.")
+	raise ImportError("[CWE-440] Python pypak Repo Failed to import.")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)